master/logging.lua

```diff
@@ -0,0 +1,17 @@
+local util = require "master.util"
+
+local state = util.mesos_get_state()
+if state == nil then
+    ngx.status = ngx.HTTP_SERVICE_UNAVAILABLE
+    ngx.say("503 Service Unavailable: invalid Mesos state.")
+    return ngx.exit(ngx.HTTP_SERVICE_UNAVAILABLE)
+end
+for _, agent in ipairs(state["slaves"]) do
+    if agent["id"] == ngx.var.agentid then
```

Comment:
we should probably make it generic so auth.lua and logging.lua can reuse the same bits. I am just not quite sure what would be the best way of doing it.

The idea here is simple, user passed a nodeID and nginx looks up in a map `mesos id = node ip` and does the reverse proxy to agent adminrouter.